rswag-specs/lib/rswag/specs/swagger_formatter.rb

```diff
@@ -199,6 +200,11 @@ def remove_invalid_operation_keys!(value)
         value.delete(:consumes) if is_hash && value.dig(:consumes)
         value.delete(:produces) if is_hash && value.dig(:produces)
       end
+
+      def remove_invalid_parameters_keys!(value)
+        is_hash = value.is_a?(Hash)
```

Comment:
The hash check is already happening on line 56 in this file, so you can drop these defensive checks.